### ISSUE TITLE
Can't save change to guest conf

### ISSUE THREAD

**Issue by frediz** [kimchi-project/kimchi#600]
I'm running kimchi from git and after defining a vm from a template, I can't save the changes that I do (can be the name of the guest or the CPU count). I initially got the same with 1.4.0 and tested with the
latest git tree.
In kimchi-access.log I get : 
9.101.4.113 - - [25/Feb/2015:04:16:31] "PUT /vms/blabla HTTP/1.0" 400 513 "https://parma.austin.ibm.com:8001/" "Mozilla/5.0 (X11; Linux i686; rv:35.0) Gecko/20100101 Firefox/35.0"
9.101.4.113 - - [25/Feb/2015:04:16:31] "PUT /vms/blabla HTTP/1.0" 400 513 "https://parma.austin.ibm.com:8001/" "Mozilla/5.0 (X11; Linux i686; rv:35.0) Gecko/20100101 Firefox/35.0"
That seems an internal error but I don't get kimchi errors in kimchi-error.log with debug settings.
Same issue when I try to change the name of template. But I don't get the same errors
Anyone has this ? I can provide access to the machine (parma.austin.ibm.com)
---
root@parma:/var/log/kimchi# uname -a
Linux parma 3.16.0-4-powerpc64le #1 SMP Debian 3.16.7-ckt4-3 (2015-02-03) ppc64le GNU/Linux
root@parma:/var/log/kimchi# dpkg -l|grep libvirt
ii  libsys-virt-perl                       1.2.9-2                               ppc64el      Perl module providing an extension for the libvirt library
ii  libvirt-bin                            1.2.9-9                               ppc64el      programs for the libvirt library
ii  libvirt-clients                        1.2.9-9                               ppc64el      programs for the libvirt library
ii  libvirt-daemon                         1.2.9-9                               ppc64el      programs for the libvirt library
ii  libvirt-daemon-system                  1.2.9-9                               ppc64el      Libvirt daemon configuration files
ii  libvirt0                               1.2.9-9                               ppc64el      library for interfacing with different virtualization systems
ii  python-libvirt                         1.2.9-1                               ppc64el      libvirt Python bindings

**Comment by cd1** [comment]
When you open the dialog "Edit VM", do you see "0" as the memory value? If you do, I guess that's the same as issue #595. You can't change any other value on that first tab when memory is set to 0. As a workaround, try reentering your memory value when editing the VM and let us know if it works. If that's not the case, then we need to investigate more.

**Comment by frediz** [comment]
Indeed, for the guest tab, that's the same problem ; setting the memory again enables to change the name too.
But in the template tab, in edition mode, I can't change anything even when changing the memory parameter which is not 0 btw...

**Comment by cd1** [comment]
Hum, I cannot reproduce the templates issue you're having. I'm running Fedora 21 and I'm able to edit the template's fields (e.g. name, memory).

**Comment by alinefm** [comment]
@frediz Could you verify it is still happening and how we can reproduce it?
